modules/tinymce/src/core/test/ts/browser/FormatterCheckTest.ts

```diff
@@ -163,6 +163,63 @@ describe('browser.tinymce.core.FormatterCheckTest', () => {
     assert.isTrue(editor.formatter.match('complex', { color: '#ff00' }), 'Selected element match with variable and function');
   });
 
+  context('using similar', () => {
+    const setupRegister = (editor: Editor) => {
+      editor.formatter.register({
+        tablecellverticalalign: {
+          selector: 'td,th',
+          styles: {
+            'vertical-align': '%value'
+          },
+          remove_similar: true
+        }
+      });
+    };
+
+    const setContentWithoutAttribute = (editor: Editor) => {
+      editor.getBody().innerHTML = (
+        '<table>' +
+          '<tr>' +
+            '<td>&nbsp;</td>' +
+          '</tr>' +
+        '</table>'
+      );
+      TinySelections.setSelection(editor, [ 0, 0 ], 0, [ 0, 0 ], 1);
+    };
+
+    const setContentWithAttribute = (editor: Editor, attributeVaue: string) => {
+      editor.getBody().innerHTML = (
+        '<table>' +
+          '<tr>' +
+            `<td style="vertical-align: ${attributeVaue};">&nbsp;</td>` +
+          '</tr>' +
+        '</table>'
+      );
+      TinySelections.setSelection(editor, [ 0, 0, 0 ], 0, [ 0, 0, 0 ], 1);
+    };
+
+    it('Without an attribute set', () => {
```

Comment:
This should include a JIRA number and it should say the expectation. eg:
```suggestion
    it('TINY-7712: should not match without an attribute set in the content', () => {
```

- [ ] Fix `it` labels to be more descriptive and include Jira numbers